vite-dev-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+yarn install
+VITE_DEV_MODE="\"docker-compose\"" VITE_FRONTEND_SERVICE_WORKER_CACHE="true" bin/vite dev
```

Comment:
Blocker: let's omit `VITE_FRONTEND_SERVICE_WORKER_CACHE="true"` from the default settings.

That variable has a very particular use case: when you're doing frontend work that requires frequent full page reloads (because frontend code you're editing isn't handled by HMR) and you want to just reuse SCP API responses (and aren't changing them).  

Such development workflows are common in JavaScript-heavy work, and this variable can save ~4-6 seconds per page load in applicable cases.  But they're not universal, and can cause frustrating cache issues -- e.g. if you cause a change in API response data and don't realize you need to clear your browser's service worker application cache.

```suggestion
VITE_DEV_MODE="\"docker-compose\"" bin/vite dev
```

I suggested a manual workaround for this setting in `vite.config.ts`; it's just a thought.  If you think it'd be better to emphasize a way to enable this non-default setting elsewhere, please do!

Reply:
Ah, thanks for the clarification - that makes sense.  I didn't do the CLI approach because `vite-dev-entrypoint.sh` is invoked from `docker-compose-dev.yaml`, and it's not possible to pass them in via the CLI.  However I might be able to read it out of the environment - lemme take a stab at that approach.

Reply:
[f4e377a](https://github.com/broadinstitute/single_cell_portal_core/pull/1641/commits/f4e377a60cb6589bcb2c71620450e9caed3914b9)